src/actions/SearchActions.ts

```diff
@@ -43,14 +43,17 @@ export const getCardSearchData = (url: string) => {
                   color_identity: info.color_identity,
                   colors: info.colors,
                   image_uris: info.image_uris,
+                  layout: info.layout,
                   mana_cost: info.mana_cost,
                   name: info.name,
                   oracle_text: info.oracle_text,
                   power: info.power,
                   rarity: info.rarity,
+                  related_cards: info.all_parts,
```

Comment:
This is a problem: normal cards do not have this attribute nor do they have transform_sides. we have to build a subtype for double faced cards

Reply:
That's what i thought at first, but we're already doing that with land cards.  Example:  land cards do not have a power/toughness so currently that property comes up as undefined.  It does not break the application.

Reply:
Yeah but I think things start getting messy when you need to figure out what kind of card it is during runtime. Especially when there are view differences between the cards. land cards and creature cards only have one face. I can use the same card view component. With transform cards i have to come up with a different because they aren't simple cards.